GOVERNANCE.md

```diff
@@ -1,53 +1,80 @@
 # Governance
 
-## Basic principle
+## Basic principles
 
-This project is governed first and foremost by reason and mutually respectful interaction between all contributors.
-The guiding approach for ensuring this is the one of [lazy consensus](https://community.apache.org/committers/decisionMaking.html).
+The Open Quantum Safe project aims to operate by the following principles:
 
-## Fallback
+- **Openness**: The project is will be open in its operation, open to contributions, and produce open source software.
+- **Respect**: The project will foster respectful interactions with all participants and contributors.
```

Comment:
participants seems to include users, contributors, committers, maintainers, right? Thus, "and contributors" appears to be superfluous.